src/Npgsql.Json.NET/JsonbHandler.cs

```diff
@@ -27,18 +28,18 @@ class JsonbHandler : Npgsql.TypeHandlers.JsonHandler
         public JsonbHandler(PostgresType postgresType, NpgsqlConnection connection, JsonSerializerSettings settings)
             : base(postgresType, connection, isJsonb: true) => _settings = settings;
 
-        protected override async ValueTask<T> Read<T>(NpgsqlReadBuffer buf, int len, bool async, FieldDescription? fieldDescription = null)
+        protected override async ValueTask<T> Read<T>(NpgsqlReadBuffer buf, int len, bool async, FieldDescription? fieldDescription = null, CancellationToken cancellationToken = default)
         {
             if (typeof(T) == typeof(string)             ||
                 typeof(T) == typeof(char[])             ||
                 typeof(T) == typeof(ArraySegment<char>) ||
                 typeof(T) == typeof(char)               ||
                 typeof(T) == typeof(byte[]))
             {
-                return await base.Read<T>(buf, len, async, fieldDescription);
+                return await base.Read<T>(buf, len, async, fieldDescription, cancellationToken: cancellationToken);
```

Comment:
The named parameter (`cancellationToken:`) here isn't needed.

I'm guessing that you added this because of my comment (https://github.com/npgsql/npgsql/pull/3158#discussion_r491334814) - the intent there was only to use named parameters when needed, because we want to not pass a previous optional parameter. In cases like this, where all parameters are specified, we can omit the parameter name.

Sorry for the extra work...

Reply:
> I'm guessing that you added this because of my comment

Yep, exactly

> Sorry for the extra work...

Nah, it's fine, I should have just asked in the first place 😅